schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
```

Comment:
- [ OPRIONAL ] I think it would be better if you make the `id` to be `NOT NULL`  and always as a primary key for several important purposes and provides numerous benefits for data integrity, indexing, and performance such as **Uniqueness**, **Data Integrity**, **Relationships**, **Performance**,... You can update it it to something like:
```sql
CREATE TABLE animals (
    id int NOT NULL,
   name VARCHAR, 
   date_of_birth DATE, 
   escape_attempts INT, 
   neutered BOOLEAN, 
   weight_kg DECIMAL,
   PRIMARY KEY (id)
);
```